### PR TITLE
Improve List docs; reformat

### PR DESCRIPTION
**Pull request by jhoerr**

### CODE REVIEW

File: src/components/List/List.md [hunk 1]
```diff
@@ -28,6 +41,9 @@
 ```
 
 ### Inline List
+
+Use the *inline* `variant` to to arrange list items horizontally (inline).
```
Comment (athill) [inline src/components/List/List.md]: Looking below, *inline* is an `orientation`, not a `variant`

File: src/components/List/List.tsx [hunk 2]
```diff
@@ -39,7 +39,8 @@ const inlineList = (orientation?: ListOrientation) => ({
 
 const componentClass = 'rvt-list';
 
-export const List : React.SFC <ListProps & React.HTMLAttributes<HTMLElement>> = ({ children, className, orientation, variant, ...props }) => {
+export const List : React.SFC <ListProps & React.HTMLAttributes<HTMLElement>> = 
+({ children, className, orientation, variant, ...props }) => {
```
Comment (athill) [inline src/components/List/List.tsx]: `...attrs` to be consistent?

File: src/components/List/List.md (hunk 1 above)
Comment (jhoerr) [reply src/components/List/List.md]: Fixed

File: src/components/List/List.tsx (hunk 2 above)
Comment (jhoerr) [reply src/components/List/List.tsx]: Fixed

**Pull request merged**